src/main/java/emissary/util/ByteUtil.java

```diff
@@ -290,20 +290,104 @@ public static String grabLine(byte[] data, int pos) {
     /**
      * Scans a byte array looking for non-printable values.
      * 
-     * @param bytes the bytes to be scanned.
-     * @return whether or not there were non-printable values.
+     * @param utf8Bytes the bytes to be scanned.
+     * @return true if the byte array contains an invalid text character.
      */
-    public static boolean hasNonPrintableValues(final byte[] bytes) {
-        boolean badCharacters = false;
+    public static boolean hasNonPrintableValues(final byte[] utf8Bytes) {
```

Comment:
Per our discussion in the office, Dave and I would like a chance to re-work this entire method and move it elsewhere.  We believe it should live in the test jar, maybe as a static member of RegressionTest or some other test utility.  I confirmed it's not used upstream in any production code.  But other members of this class are used in production code, so we should leave the class and it's other methods here.